### PR TITLE
fix: 500 when unknown user id

### PR DESCRIPTION
**Pull request by Aleksgav**

### CODE REVIEW

File: controllers/account.go [hunk 1]
```diff
@@ -43,6 +43,11 @@ type Response struct {
 	Data2  interface{} `json:"data2"`
 }
 
+type Error struct {
```
Comment (ISulimanov) [inline controllers/account.go:46]: move to base.go

File: controllers/util.go [hunk 2]
```diff
@@ -49,6 +50,21 @@ func (c *ApiController) ResponseError(error string, data ...interface{}) {
...
 }
 
+// ResponseErrorWithStatus ...
+func (c *ApiController) ResponseErrorWithStatus(status int, error string) {
+	resp := &Error{Code: status, Message: error}
+
+	http.Error(c.Ctx.ResponseWriter, error, status)
+
+	c.Data["json"] = resp
+	c.ServeJSON()
```
Comment (dplynskiy) [inline controllers/util.go:60]: is it works right? looks like http.Error and c.ServerJSON do same work, mb u.Ctx.Output.Status = http.StatusNotFound instead http.Error will better here?

Comment (Aleksgav) [reply controllers/util.go:60]: yep, it's a better way thx

File: conf/app.conf [hunk 3]
```diff
@@ -2,8 +2,8 @@ appname = casdoor
 httpport = 8000
 runmode = dev
 copyrequestbody = true
-driverName = mysql
-dataSourceName = root:root@tcp(localhost:3306)/
+driverName = postgres
+dataSourceName = "user=alexg password= host=localhost port=5432 sslmode=disable dbname=casdoor"
```
Comment (dplynskiy) [inline conf/app.conf:6]: del changes from commit

**Review by dplynskiy — APPROVED** [review]

**Review by ISulimanov — APPROVED** [review]

**Pull request merged**